client/public/index.html

```diff
@@ -9,10 +9,6 @@
       name="description"
       content="Web site created using create-react-app"
```

Comment:
... and could change the description

Reply:
Perfect, thank you for pointing out that I already wanted to do that. 😅